### PR TITLE
Building on flow

### PR DESCRIPTION
**Pull request by nialexsan**
https://docs-git-building-on-flow-onflow.vercel.app/next/building-on-flow

this PR implements a new docs structure for `building on flow` section according to [notion](https://www.notion.so/dapperlabs/Dev-Documentation-breakdown-28be7d292dee4049a3db43e457a57151?pvs=4#9b1a49e36c58487b907d69c8d4bb5d03)

Screenshot 2023-08-17 at 17 28 47

### CODE REVIEW

**Comment by chasefleming** [comment]
There's no description or issue linked. Can you explain what one should be looking for here?

**Comment by chasefleming** [comment]
Will this go live with the missing content if it's merged?

**Comment by nialexsan** [comment]
yes, it will, that's why it's a draft
I think you can branch off this PR and when everyone finishes working on this section, we can merge it

alternatively I can remove top bar menu item and merge it
this section will be kinda hidden without

File: docs/guides/_category_.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
   "label": "Guides",
-  "position": 1
+  "position": 7
```
Comment (nialexsan) [inline docs/guides/_category_.json:3]:
```suggestion
  "position": 1
```

**Pull request merged**